neptune/experiments.py

```diff
@@ -323,21 +342,24 @@ def properties(self):
 
             Get experiment properties.
 
-            >>> experiment.properties
+            >>> experiment.get_properties
 
         """
-        return self._simple_dict_to_dataframe(self._leaderboard_entry.properties)
+        experiment = self._client.get_experiment(self.internal_id)
+        return dict(
```

Comment:
Should we return dict or DataFrame here?

Reply:
Fixed